exodus_core/analysis/static_analysis.py

```diff
@@ -156,10 +156,10 @@ def load_trackers_signatures(self):
         """
         self.signatures = []
         exodus_url = "https://reports.exodus-privacy.eu.org/api/trackers"
-        with urllib.request.urlopen(exodus_url) as url:
-            data = json.loads(url.read().decode())
-            for e in data['trackers']:
-                self.signatures.append(namedtuple('tracker', data['trackers'][e].keys())(*data['trackers'][e].values()))
+        r = requests.get(exodus_url)
+        data = json.loads(r.text)
```

Comment:
Why not `data = r.json()` directly ?
We could get read of the json python module in the same time.

Reply:
What a good idea. I was focusing on how to replace each step of the urllib library one after one. I should fix it.